src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -5,3 +5,20 @@
 (defn get-first-name
   [full-name]
   (first (string/split full-name #" ")))
+
+(defn get-balance-by-address
+  [balances address]
+  (->> balances
+       (filter #(= (:address %) address))
+       first
+       :balance))
+
+(defn get-account-by-address
+  [accounts address]
+  (->> accounts
+       (filter #(= (:address %) address))
+       first))
```

Comment:
Instead of using `filter` + `first` we could use `some`

Reply:
Done